Examples/Tests/laser_injection_from_file/inputs.1d_boost_test

```diff
@@ -0,0 +1,59 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+stop_time = 2.45e-12
+amr.n_cell = 1024
+amr.max_grid_size = 512
+amr.blocking_factor = 32
+amr.max_level = 0
+geometry.dims = 1
+geometry.prob_lo     = -25.e-6   # physical domain
+geometry.prob_hi     =  25.e-6
+warpx.verbose = 1
+warpx.serialize_initial_conditions = 1
+
+# Boost parameters
+warpx.gamma_boost = 10
+warpx.boost_direction = z
+warpx.do_moving_window = 1
+warpx.moving_window_dir = z
+warpx.moving_window_v = 1.0
+
+#################################
+####### Boundary condition ######
+#################################
+boundary.field_lo = pec
+boundary.field_hi = pec
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.cfl = 0.98
+warpx.use_filter = 0
+algo.maxwell_solver = ckc
+algo.load_balance_intervals = -1
+
+# Order of particle shape factors
+algo.particle_shape = 3
+
+#################################
+############# LASER #############
+#################################
+lasers.names        = lasy_laser
+lasy_laser.position     = 0. 0. 0. # This point is on the laser plane
+lasy_laser.direction    = 0. 0. 1.     # The plane normal direction
+lasy_laser.polarization = 0. 1. 0.     # The main polarization vector
+lasy_laser.e_max        = 1.e14        # Maximum amplitude of the laser field (in V/m)
+lasy_laser.wavelength = 1.0e-6         # The wavelength of the laser (in meters)
+lasy_laser.profile      = from_file
+lasy_laser.time_chunk_size = 100
```

Comment:
Could you use this, along with the changes from https://github.com/ECP-WarpX/WarpX/pull/4184
```suggestion
lasy_laser.time_chunk_size = 50
```